UnityProject/Assets/Scripts/Objects/NukeInteract.cs

```diff
@@ -46,8 +46,6 @@ IEnumerator WaitForDeath()
 		GibMessage.Send();
 //		GameManager.Instance.RespawnAllowed = false;
 		yield return new WaitForSeconds(2f);
-		//Restart Round:
-		GameManager.Instance.RestartRound();
```

Comment:
did you test and see what happens when this is removed?
it looks like this way round never ends after nuke is detonated...